6220 Autonomous Program

cswelch opened the pull request:
This is 6220's autonomous program after having merged branches.

eukota commented:
This is probably an erroneous pull request. blah151 - you have control over this so there is no need to do a pull request.

rgatkinson committed:
remove dead ibActionFlag constant; remove MODE_CACHE_STATUS

rgatkinson committed:
remove dead ibActionFlag constant; remove MODE_CACHE_STATUS


Former-commit-id: bf92c41fbefa40aad2229e33f1065704286b0829
Former-commit-id: 8966ac03d78ee07081c376de5502e3bb96dafca2

rgatkinson committed:
rename I2cDeviceClient -> I2cDeviceSynch


Former-commit-id: b12856a8a9eca90746b1cb86006a4cfab4a997a8
Former-commit-id: 3dabb11a4a1f9f82bbc90fb823df6ad689d47892

rgatkinson committed:
rename I2cDeviceClient -> I2cDeviceSynch

rgatkinson committed:
move I2cDeviceSync interface to SDK


Former-commit-id: 724c8ec337e650ed797f41dd59684c42e9e4a8f3
Former-commit-id: 6537cf370331d6283e8588370dc880b438fb0ea4

rgatkinson committed:
move I2cDeviceSync interface to SDK

rgatkinson committed:
I2CDeviceSynchImpl -> I2cDeviceSynchImpl


Former-commit-id: 7752f900e443087c1f347e980adcd8a84a39c61b
Former-commit-id: bb37bd3ca3d545be23e367ed6c03062d293d3ac3

rgatkinson committed:
I2CDeviceSynchImpl -> I2cDeviceSynchImpl

rgatkinson committed:
remove dependency on Util

rgatkinson committed:
remove dependency on Util


Former-commit-id: eba0d74aeadb9bc21fa4c1f3cb8ed35273f9cc59
Former-commit-id: a19375217ff5cef521f8ffe2169e46043a2ad4e5

rgatkinson committed:
non swerve logging tag

rgatkinson committed:
non swerve logging tag


Former-commit-id: 0ff99bd5a0dde28700460906be1bf068bbddafe7
Former-commit-id: cfe6954bfc689185176011edb3c415b7fe1e279a

rgatkinson committed:
simplify the way we do asserts: assert removal in release builds will be something for ProGuard to deal with.


Former-commit-id: a89c87b008d1397fada145abbd726fc5281725e9
Former-commit-id: 4a2664d488aaab476b7753977d92aeebb6d03310

rgatkinson committed:
simplify the way we do asserts: assert removal in release builds will be something for ProGuard to deal with.

rgatkinson committed:
guard against pretend mode


Former-commit-id: 791a0c6985797e042e42e5afaf2b8c81c0a6b244
Former-commit-id: 6ff5d748496655b9550fee0192f3b85b9eeea6f2

rgatkinson committed:
guard against pretend mode

rgatkinson committed:
rename a couple of variables for clarity

rgatkinson committed:
rename a couple of variables for clarity


Former-commit-id: 196e83c0299baa6f6bceb19567543a94db38f10e
Former-commit-id: 8b71c65dd44765dc84b342a8531ccc42cfcfe1a3

rgatkinson committed:
remove last swerve dependence


Former-commit-id: 351a2eec297a0c4a034a7438a91f11331450643f
Former-commit-id: 402657ee3a0e6c311d31a6243b910c5392861850

rgatkinson committed:
remove last swerve dependence

rgatkinson committed:
remove auto-close functionality


Former-commit-id: 77107c9027db435728e4d0fb116c6e1784b39f3d
Former-commit-id: 43bfb3359acaf9b54ef3e290ef179fa8df3a73a0

rgatkinson committed:
remove auto-close functionality

rgatkinson committed:
remove yet more swerve dependence


Former-commit-id: 8fb0615a4ff3529c1bdd91456300096dc35de7f5
Former-commit-id: 4b7c7d52251ad45dcddd60ac304814b7bf27ca6c

rgatkinson committed:
remove yet more swerve dependence

rgatkinson committed:
move i2cdevicesyncimpl to sdk

rgatkinson committed:
move i2cdevicesyncimpl to sdk


Former-commit-id: f1ad4d6ad7b05d87fabe35778693e654be884afb
Former-commit-id: d8a02d5af445b9ea37140fdd6d4f31d2f0d7497d

rgatkinson committed:
fix EasyLegacyMotorController.setMotorChannelMode bug when using RESET_ENCODERS

rgatkinson committed:
fix EasyLegacyMotorController.setMotorChannelMode bug when using RESET_ENCODERS


Former-commit-id: d8ae3b9c09fbbcbb8ddf3d4080ec4d420c83edc3
Former-commit-id: b1499b2f7808fe07a58a3c24c378b1d84bff396b

rgatkinson committed:
merge master -> synchronous


Former-commit-id: 3702773196886bb013ac7afef0cbbbac228803a2
Former-commit-id: de16be457bf237b3256188b82ea48e33ce9423b5

rgatkinson committed:
merge master -> synchronous

rgatkinson committed:
variable rename

rgatkinson committed:
variable rename


Former-commit-id: 4b0fd2aa6313275eb58e81c7adc6dfd4f9c05769
Former-commit-id: 21b27b56c231d496c9b33c9e0ae32307fdc47634

rgatkinson committed:
dont use easy controller explicitly


Former-commit-id: 213faaaafef393e28337f9509bda23ff96cc642a
Former-commit-id: f9e25b81c80732fd9b53d948c442223e147eb559

rgatkinson committed:
dont use easy controller explicitly

rgatkinson committed:
comments, deprecation


Former-commit-id: cf79db90f35ece87263fcaf77faf2c669c333bfc
Former-commit-id: 568aff4c093ac34ccd75dbd1e7ce1135bcfd0dc1

rgatkinson committed:
comments, deprecation

rgatkinson committed:
remove and put back his voltage sensor


Former-commit-id: f0ecedc10263f24de67b3d3a7c6d84cb60e1679f
Former-commit-id: ecd63428fa094bf9af007fd1b62af25b222fb7e6

rgatkinson committed:
remove and put back his voltage sensor

rgatkinson committed:
simpler logic


Former-commit-id: ec2c5a34a3a9e6f9d610d747bb0285fc01920366
Former-commit-id: 58a7ea2fcdd473bee47379d242dd9d38a30b7e84

rgatkinson committed:
simpler logic

rgatkinson committed:
necessary for previous commit


Former-commit-id: d688ab1c7ddd6a8eeda693201ac4a0fafb3a72c0
Former-commit-id: dc7b070915ef5ab42644fc9cdfa61c5728a51751

rgatkinson committed:
necessary for previous commit

rgatkinson committed:
don't switcheroo legacy motor and servo controllers


Former-commit-id: 63fa21c7693f87faaef558e27bb019dd7864ac2a
Former-commit-id: 856e37c48551ba08731d212b247fbe2b8ecc8081

rgatkinson committed:
don't switcheroo legacy motor and servo controllers

rgatkinson committed:
use OpModeServices


Former-commit-id: 175ae2f25134bc102337ef69da9e5b70100048a5
Former-commit-id: 5e4c4a370caaf6bb13c772bbf5bbc2b6fa30d64c

rgatkinson committed:
use OpModeServices

rgatkinson committed:
comments and JavaDoc; stop using all Easy* objects


Former-commit-id: f78aa4425bf4c85537f3506b77d757e3fdc10afb
Former-commit-id: abbd2b5caf8e3a422fb9b923ee580d286eb6cec5

rgatkinson committed:
comments and JavaDoc; stop using all Easy* objects

rgatkinson committed:
renaming to reduce usage of 'thunk', comments; remove unthunkedHardwareMap

rgatkinson committed:
renaming to reduce usage of 'thunk', comments; remove unthunkedHardwareMap


Former-commit-id: 153fde56edcdd60cd774625aad067306996925d9
Former-commit-id: dbad3a5b085420e94331f222b46dc7ef1c5ee017

rgatkinson committed:
remove easymodern contoroller instantiation


Former-commit-id: 79b98a74602bedbc1790eef8a23f982aeca87e3d
Former-commit-id: 98814bef837979ce0f50af70fd6ab29614d1912f

rgatkinson committed:
remove easymodern contoroller instantiation

rgatkinson committed:
more cleanup

rgatkinson committed:
more cleanup


Former-commit-id: 63e662c428b7c6869c9c42224acd8abd9ca1b906
Former-commit-id: 5459b90452d44ac28c203671ce309d34797c7604

rgatkinson committed:
don't thunk DeviceInterfaceModule or LegacyModule

rgatkinson committed:
don't thunk DeviceInterfaceModule or LegacyModule


Former-commit-id: 8a12916d8c34c0c908418a56fe761f6621e14ceb
Former-commit-id: d0b37d2131e95603c9688783b3ef50daea2abc39

rgatkinson committed:
fix TelemetryDashboardAndLog ctor usage in SynchronousOpMode; improve motor servo tests

rgatkinson committed:
fix TelemetryDashboardAndLog ctor usage in SynchronousOpMode; improve motor servo tests


Former-commit-id: 6d1d2f24b1fdfb5796a6c8a2e381fce2a5eb57bb
Former-commit-id: 44f76c639a6295113c3730eebcc5fdec57283f3d

rgatkinson committed:
bump app version; restore compabibiliyt

rgatkinson committed:
bump app version; restore compabibiliyt


Former-commit-id: 7b4c51c9bf7bdf00d5f7e740ba17a6df1c1462d7
Former-commit-id: 53401c5022f1ac7e44da95af2167e24d691b2083

rgatkinson committed:
comment


Former-commit-id: fd00392def33738157c46c4b4487c975cf5e28c8
Former-commit-id: ba7f839336ee37cb03b53829dd0c65e45b6d60d0

rgatkinson committed:
comment

rgatkinson committed:
track sdk


Former-commit-id: 8fb06d4dd70a251a233e0572e76b352b301b06a5
Former-commit-id: 5e4f4e282a3a57a2098bc6e0a8c30665ee277308

rgatkinson committed:
track sdk

rgatkinson committed:
remove dead thunkers to avoid having to upgrade them; more telemetry and fixed telemetry in SyncTeleOp; cut down on gamepad tracing


Former-commit-id: 4eb1eaf50fb33f50917d3fdbb71edb7d268a1825
Former-commit-id: a8019d50a98d3d71330bacbf99519c093537e4a3

rgatkinson committed:
remove dead thunkers to avoid having to upgrade them; more telemetry and fixed telemetry in SyncTeleOp; cut down on gamepad tracing

rgatkinson committed:
remove dead classes


Former-commit-id: 317a4d47a2eee1046f5d05ecaef87099c0cb95b5
Former-commit-id: 7cfe5060aef6b880d09067f84aacb03b2698f754

rgatkinson committed:
remove dead classes

rgatkinson committed:
update example opmodes, add more addData overloads to telemetry


Former-commit-id: 3f5887081881e5e5c220dcaaa58e2661e3c982f7
Former-commit-id: c7fab07eb7cb48b4734cc6fbc6ab0b38df372f6c

rgatkinson committed:
update example opmodes, add more addData overloads to telemetry

rgatkinson committed:
nuke HandShakeThreadStarter

rgatkinson committed:
nuke HandShakeThreadStarter


Former-commit-id: 9cc0760f77d98b83d6cff16bff5593e14b3a9b24
Former-commit-id: b4e77f5b8a3a95184e5239a27f916de81c7a3611

rgatkinson committed:
submit vs execute


Former-commit-id: 2131a4ddb44b3d0abe67ab2764f6517dd0efb2b3
Former-commit-id: 80bfb4a3018e44e44ce7151d6802971ae2a877ee

rgatkinson committed:
submit vs execute

rgatkinson committed:
track sdk; update version

rgatkinson committed:
track sdk; update version


Former-commit-id: 6e3bae49caf998626c3d992661d8bd5af665aeb9
Former-commit-id: d9f582b856ad341d0c270b6316086a6088eee9f1

rgatkinson committed:
capture corresponding AARs


Former-commit-id: 4b9fb32fa55b03ea7e575eb1c4576401fc377fa0
Former-commit-id: be839c7ba94af2823e6cbcc4207373a9d697779d

rgatkinson committed:
capture corresponding AARs

rgatkinson committed:
track thread-safety enhancements in SDK; reduce local paranoia

rgatkinson committed:
track thread-safety enhancements in SDK; reduce local paranoia


Former-commit-id: e9c9251d88a6def20a15c7b535bd30fa0305a775
Former-commit-id: fc6364913034a838b7789175d8d2940f6c394c9f

rgatkinson committed:
remove thunking wrapper classes

rgatkinson committed:
remove thunking wrapper classes


Former-commit-id: f244d13e4375959b1ef411e3d1c9b4f0c1698643
Former-commit-id: b43788edcc32042e8c6cd240d033bb5bfe2ccd89

rgatkinson committed:
remove 'action key' notion; begin to clean up thunking infrastrructure

rgatkinson committed:
remove 'action key' notion; begin to clean up thunking infrastrructure


Former-commit-id: fb6090afbb8876541cb1fe88dcc2154bcb7efad9
Former-commit-id: da9259b921cafe9ffe5027535842e2bf5d21ed3e

rgatkinson committed:
auto-stop synchronous opModes

rgatkinson committed:
auto-stop synchronous opModes


Former-commit-id: 589ca95b528472235a28dad21512407ab4589e01
Former-commit-id: 615c2b549c21daaa760f630dcfe49426777e328b

rgatkinson committed:
add waitForStart()!

rgatkinson committed:
add waitForStart()!


Former-commit-id: 33777bef3009bb309ef4be3a649c6617d4fcc406
Former-commit-id: 0165fe74ea377b1c2d82516b96b42951a9f2e5a0

rgatkinson committed:
add AARs


Former-commit-id: 4ebc029e6eaf67d1ab8bf08aac394b4ee3875268
Former-commit-id: 8a27a97bc56c54efcaf8a3a712b38ab5e84353af

rgatkinson committed:
add AARs

rgatkinson committed:
power can't be set while in reset encoders; don't need to manually kill power on exit; break out of loop early if opmode stops!

rgatkinson committed:
power can't be set while in reset encoders; don't need to manually kill power on exit; break out of loop early if opmode stops!


Former-commit-id: 1d54fe570c40e7e9fd76285c59f3e2a5b3c93f11
Former-commit-id: f801d217cbe19841011eb44fba4b876cf594b3a1

rgatkinson committed:
deal with cancellationexception

rgatkinson committed:
deal with cancellationexception


Former-commit-id: 5847743686b6512ada5ff695120d3761328fad24
Former-commit-id: 8ffd0a14f442c30aa958f8fdbebe45a075c04965

rgatkinson committed:
Merge branch 'synchronous'

rgatkinson committed:
Merge branch 'synchronous'


Former-commit-id: cfc87dc33f7e36c17799d62396d6384faaeb606f
Former-commit-id: d60732bb3916dcd2d738c3eb633e626c5f52306a

rgatkinson committed:
Merge branch 'threadSafe'


Former-commit-id: 0b6601c6978b3860e8d41f426f1768d45294fb4b
Former-commit-id: f49ad89667501201566f2d12c604f5b458b9f8c8

rgatkinson committed:
Merge branch 'threadSafe'

rgatkinson committed:
add test


Former-commit-id: 231f5925d638685930518b98514938b52951f81e
Former-commit-id: d483398048fa5d161da145e6a1fe6e9f7f4ec66e

rgatkinson committed:
add test

rgatkinson committed:
Merge branch 'synchronous'

rgatkinson committed:
Merge branch 'synchronous'


Former-commit-id: 86a0e17708f6ac65c102969fd2a796f0bd186756
Former-commit-id: bcc96527a0c4efd7503a0eae997467e9eab81bc0

rgatkinson committed:
Merge branch 'threadSafe'


Former-commit-id: 49b3136bb62e1198f45fac85c8f3f3a4b70564f6
Former-commit-id: 6c198a16e7ed9d19f061e4bbae055820cba518c6

rgatkinson committed:
Merge branch 'threadSafe'

rgatkinson committed:
mrege


Former-commit-id: 798b52bbc0894b0a54a61e4b2e12c57ea363177b
Former-commit-id: e7e35946095a231cbb36fb0115db441e46ae7a0b

rgatkinson committed:
mrege

rgatkinson committed:
add tests

rgatkinson committed:
add tests


Former-commit-id: 71d0f014b9846c24b559927878fd23e495403e0b
Former-commit-id: 532ef01842bf194afc846af4f413ee4a2313478b

rgatkinson committed:
add TimingTheTimer

rgatkinson committed:
add TimingTheTimer


Former-commit-id: b802f0a42e2ebf512012f2c11711e76d45b1f8e4
Former-commit-id: d6ae736e8ec67c3cb63a8072d88fc5bf3194398a

rgatkinson committed:
add InfiniteLoopTest


Former-commit-id: a87bd66bacf83e029babacf32964548e4c28a114
Former-commit-id: aa769324ddb6782fc29f451297b21d439eb7d86f

rgatkinson committed:
add InfiniteLoopTest

rgatkinson committed:
Update README.md

rgatkinson committed:
Update README.md

Former-commit-id: e22ca241e50a0362e77dc0c4157e155359d1a16e
Former-commit-id: 670d249cd103dc6e29e87d0ab60dacfcd1ee2de8

rgatkinson committed:
Update README.md

Former-commit-id: e7c8e70f56166b75e7515427846cb2e8dd117e2e
Former-commit-id: f12b3a4e29fe783436884415bf51550d2ef9d908

rgatkinson committed:
Update README.md

rgatkinson committed:
add gamepad latency test


Former-commit-id: 89194baa9ebecbfd925d837f7c4c1fd0f5e0593d
Former-commit-id: f8d4a8818632d6fa3729fbdccec1b4f2c5fc591b

rgatkinson committed:
add gamepad latency test

rgatkinson committed:
add InfiniteLoop


Former-commit-id: 51804a179c97f07643553a1f00353f10ee382dda
Former-commit-id: c99af01f61755639b3806314d2cd9c81de3d6962

rgatkinson committed:
add InfiniteLoop

rgatkinson committed:
READ_MODE -> ReadMode


Former-commit-id: 7c0ff09dc3210570b47df8d00f35bd4736bad97c
Former-commit-id: 6ab1f6364c9cd8d6a64b150c834897d5a614ffe4

rgatkinson committed:
READ_MODE -> ReadMode

rgatkinson committed:
track sdk changes

rgatkinson committed:
track sdk changes


Former-commit-id: 1fac9ae16064edf1af4f050d37e9b538b9c5ee64
Former-commit-id: a290ef35c81a6630781c06f706261e4da60696b6

rgatkinson committed:
Merge branch 'synchronous'

rgatkinson committed:
Merge branch 'synchronous'


Former-commit-id: 2a4c68f2d39d91cbda04661139db1d7bc048553e
Former-commit-id: 14358e030b5604de26646d55a276abaab074fe75

rgatkinson committed:
Merge branch 'threadSafe'

rgatkinson committed:
Merge branch 'threadSafe'


Former-commit-id: fab0ebb78df5bdcd57182d76045d75278bc4cbcf
Former-commit-id: 339c775006a9bb6714f1ee39fb8c1350b6767664

rgatkinson committed:
merge


Former-commit-id: 9f27fcd39632dbccb1d360e153c52b7df6fa500d
Former-commit-id: d0284e9dc26e34cfeac2df3ea3631c676913c6f9

rgatkinson committed:
merge

rgatkinson committed:
disable test by default


Former-commit-id: adb50df37ad0b47e787ffceb5f5405d57ddb6374
Former-commit-id: 1257161339a2e0a9ea810138ebce2928c76ea441

rgatkinson committed:
disable test by default

rgatkinson committed:
use executors for synchronous worker threads so the shutdown logic will help us find deadlocks


Former-commit-id: fd9725beb1c15f90ffb1069f031f1413981fa340
Former-commit-id: 324dfa5b5293c7fda3322b2728e2c5adbf433ca9

rgatkinson committed:
use executors for synchronous worker threads so the shutdown logic will help us find deadlocks

rgatkinson committed:
add d2xx documentation

rgatkinson committed:
add d2xx documentation


Former-commit-id: bf2815aa8f69f90515677d9f94cc32dcb69eea2b
Former-commit-id: ec8e143749210a323daca847be435a33933361d4

rgatkinson committed:
be more robust about class loader errors


Former-commit-id: 985cc2a7c9159ca71ee976c367b35106953e64f6
Former-commit-id: 7739c734438efadc98af4509e69b4ea0e92fc8fc

rgatkinson committed:
be more robust about class loader errors

rgatkinson committed:
Merge branch 'master' of https://github.com/SwerveRobotics/ftc_app.git

rgatkinson committed:
Merge branch 'master' of https://github.com/SwerveRobotics/ftc_app.git


Former-commit-id: 1eb18c2fc52ac14627391618df7ad1628bca82e7
Former-commit-id: 4a3e0396ed006df59cddbffb51a7b35e5e4c1756

rgatkinson committed:
Merge branch 'MentorSandbox'

rgatkinson committed:
Merge branch 'MentorSandbox'


Former-commit-id: 827e0fb5c830690ab67699ba4be50910a22f8219
Former-commit-id: 9f89d884c135220ea2702e852e622ff67f5338ed

rgatkinson committed:
remove more remnants of thunking

rgatkinson committed:
remove more remnants of thunking


Former-commit-id: 003d734a32911881a984ee9f651c6bcb9514f50f
Former-commit-id: ddd7555daff77f59b063f114e980e0f657b1f81e

rgatkinson committed:
cleanup executor shutdown logic

rgatkinson committed:
cleanup executor shutdown logic


Former-commit-id: 7db99565180e55722341401fa236e2fc3ed98ce0
Former-commit-id: b996368478c920c570070d2b9302c4fb52eb74a8

rgatkinson committed:
update version


Former-commit-id: 084b257302bfa2aa21fafa6b98eb3fae759ca110
Former-commit-id: bbfce9d6dc7458f6a913ca82a680129c59b93108

rgatkinson committed:
update version

rgatkinson committed:
add univeral debug signing key

rgatkinson committed:
add univeral debug signing key


Former-commit-id: 362f82fa3b6fcdc59513da98c1541be5c8cf4837
Former-commit-id: 002834c018393a16b01734b988ad17fcfbb2df1f

rgatkinson committed:
Remove RuntimeInterruptedException in favor of CancellationException

rgatkinson committed:
Remove RuntimeInterruptedException in favor of CancellationException


Former-commit-id: b8996304f61d25b695ad878a28ef41ec2d507db1
Former-commit-id: 77e68902be1bede8983f5bbb3f1160782894f04f

rgatkinson committed:
update binaries


Former-commit-id: 50577ead210eb78e7220ef8ea9bf1aba97fed10a
Former-commit-id: 1c669298c9bfd98664df1115f3a9e034c24ad6f1

rgatkinson committed:
update binaries

rgatkinson committed:
update driver station


Former-commit-id: 04547a5c802bcd5072ac5f6ac62e65380ba7218f
Former-commit-id: 3a7d7e82bcbefb616bf01dc18226a0c1b37fd3b9

rgatkinson committed:
update driver station

rgatkinson committed:
update version numbers

rgatkinson committed:
update version numbers


Former-commit-id: 29b2a83e23d1ca263b0bb2540982b6139c90ce43
Former-commit-id: 9a516550a1947e2d5ef18f4d4997c06a3cc4fd05

rgatkinson committed:
update readme


Former-commit-id: 9331e8b8ff93b790e200ed1a8fc3fef833e96b8b
Former-commit-id: fe672851f6e1922a5a5022f5113be79fd126ae2b

rgatkinson committed:
update readme

rgatkinson committed:
more robust monitoring


Former-commit-id: 0b8e7aa08f14ded02d24b4be990ee46a2e1d9b65
Former-commit-id: 7451c134ba11fd31f60481ecc41370ebea4e5c3a

rgatkinson committed:
more robust monitoring

rgatkinson committed:
changes after review

rgatkinson committed:
changes after review


Former-commit-id: f6ec76ae001ec3610b3608141ac2d2460dd94ede
Former-commit-id: 0f8fef69780e2388dbda8e5fc9d4706b4a20b917

rgatkinson committed:
avoid I2cDevice leakage

rgatkinson committed:
avoid I2cDevice leakage


Former-commit-id: b14028fa6f817bef7e03f9da1d4346385a0a8355
Former-commit-id: d96ad056c5b374efab20ef486ecac7710ba365ce

rgatkinson committed:
Regularize example opmode display names. Adjust sort order of opmode list on driver station.

rgatkinson committed:
Regularize example opmode display names. Adjust sort order of opmode list on driver station.


Former-commit-id: e7ce679f3ce34aefe5cae377885c59423e2a8c71
Former-commit-id: 13a1c6424a4a96641624ef750e2779c8a5a8fcdb

rgatkinson committed:
Regularize names of example opmodes

rgatkinson committed:
Regularize names of example opmodes


Former-commit-id: 3790b791863eba0d73fc43eb8dd5e61a5f8de6c5
Former-commit-id: 6d1e1842027cd6dae732f15e150b945f71948c48

rgatkinson committed:
lift common sound files to swerve library from yourcodehere


Former-commit-id: 40856491c694839707e0017fd93d2e46ed015bd0
Former-commit-id: f58be53202ce1be1398b08ccf05b3ba6aa169113

rgatkinson committed:
lift common sound files to swerve library from yourcodehere

rgatkinson committed:
updates after review

rgatkinson committed:
updates after review


Former-commit-id: 12613f63bd9f7bee767fd6ef7fce0a6fb9788aee
Former-commit-id: 796a5400bf63922614e81e126a49aea6ed478dfd

rgatkinson committed:
update driver station w/ new version #

rgatkinson committed:
update driver station w/ new version #


Former-commit-id: 6a2617c6a64c35d35abad02cdacf3cf9083c79fc
Former-commit-id: bfb94c4e24eb6e0f8589eb3c20a08b4a5e7a2af7

rgatkinson committed:
Mergve Swerve.1.5.0 branch into master


Former-commit-id: 62e4cd4caded722512111f7157678ef3a697a539
Former-commit-id: c511fd29b258d1186de32dd7b0e2d1ab514a30c1

rgatkinson committed:
Mergve Swerve.1.5.0 branch into master

nrizzio committed:
Mark documentation directories as such

This way, files in those directories (ex. html for javadoc) no longer count towards the overall repository language percentage.

nrizzio committed:
Mark documentation directories as such

This way, files in those directories (ex. html for javadoc) no longer count towards the overall repository language percentage.

Former-commit-id: 78f51bf08faf65b1ea555b0c908404ef7601b370
Former-commit-id: 423f659b9e345aa51a5f9923beab2be28b613f46

ftctechnh committed:
Added v1.7 (with library 16.03.09) driver station and robot controller APK files.

ftctechnh committed:
updated javadoc for library built on 16.03.09.

ftctechnh committed:
removed ftc_app.iml and FtcRobotController.iml files from repo.

ftctechnh committed:
Update .aar files with new versions from 16.03.09. Update FtcRobotController/build.gradle to use compileSdkVersion 19. Added FtcRobotController/libs/ftc.debug.keystore so all user generated apps are signed by the same key. Update AndroidManifest.xml with current version number.

ftctechnh committed:
Updated README.md file to reflect 16.03.09 changes.

rgatkinson committed:
new binaries

rgatkinson committed:
new binaries


Former-commit-id: 623d659367ea59750f2c8f06b40db6a3d2954e1e
Former-commit-id: d87028392cbe476a68382fa1e768a24dfc4b01f9

rgatkinson committed:
update binaries

rgatkinson committed:
update binaries


Former-commit-id: d545840d9322f68c826e59610c0abf7052574d7f
Former-commit-id: e3fdf563bb05e401f81b86e0423e03fcb80fb0e6

rgatkinson committed:
desynchronize FtcEventLoop methods


Former-commit-id: e59583dc6d827e75e2b748e5e08603db7180e5b0
Former-commit-id: d40c284cf23bc98939f5c492edcc08801a781853

rgatkinson committed:
desynchronize FtcEventLoop methods

rgatkinson committed:
desynchronize OpModeManager


Former-commit-id: da79452c2103da44dfbfebf00bc67a6ddf84d374
Former-commit-id: 83c40fc1404cd686964838f84119b7814ac08cb0

rgatkinson committed:
desynchronize OpModeManager

rgatkinson committed:
more robust opmode init

rgatkinson committed:
more robust opmode init


Former-commit-id: 7ef1b1ced52a767b0ab8b740d35ef37aa8f8be4e
Former-commit-id: 2e62c15b4a3630062741d9867da9764844122975

ftctechnh committed:
updated .apk files to v1.77 versions

ftctechnh committed:
updated javadoc

ftctechnh committed:
updated FtcCommon, Hardware and RobotCore AAR files with 16.03.30 versions.

ftctechnh committed:
updated README.md file.

rgatkinson committed:
add external specs


Former-commit-id: 8a5aa0d7d435a3129792f2a38d9d49ea83ffe8f5
Former-commit-id: 04b07201660d51b5486f62401573a343806c73ee

rgatkinson committed:
add external specs

rgatkinson committed:
Merge branch 'master' of https://github.com/SwerveRobotics/ftc_app.git


Former-commit-id: 89d972ffea603308b8ba0eb3f0af1da52dc8f119
Former-commit-id: 4623102c94ba09c70e0635c8e7b29750a60901c1

rgatkinson committed:
Merge branch 'master' of https://github.com/SwerveRobotics/ftc_app.git

rgatkinson committed:
Merge pull request #86 from nrizzio/mark-doc-directories

Stop treating files in documentation directories as code

Former-commit-id: e2eed217916a8974796c0341f45ee0ea91976e23
Former-commit-id: 3e4cc7744001cbcfbdb632ff7bc6782bf29cef8a

rgatkinson committed:
Merge pull request #86 from nrizzio/mark-doc-directories

Stop treating files in documentation directories as code

rgatkinson committed:
merge changes from master: new spec sheets, updated .gitattributes

rgatkinson committed:
merge changes from master: new spec sheets, updated .gitattributes


Former-commit-id: 820991af824ed2ca29bdb5b36f725807215ae90c
Former-commit-id: ee44d0e7dde32ac236c8b8cb2999e167a593b285

rgatkinson committed:
merge 1.5.2.beta++ -> master

rgatkinson committed:
merge 1.5.2.beta++ -> master


Former-commit-id: 32c6f43f66bce1213a3e237a2ef6f4599746a4f5
Former-commit-id: 41905f8bbab47b852dd009cf1ed70cbe24d51569

rgatkinson committed:
Upgrade gradle-wrapper.propertis and classpath (now com.android.tools.build:gradle:2.1.2) per Android Studio 2 prompts. Remove unneeded android-support-v4.jar references.


Former-commit-id: fdd7718f5d3125999024b8ed2a7e52df137f0aa8
Former-commit-id: 1089ecba4effce55b0d54924c5554d0b739d810a

rgatkinson committed:
Upgrade gradle-wrapper.propertis and classpath (now com.android.tools.build:gradle:2.1.2) per Android Studio 2 prompts. Remove unneeded android-support-v4.jar references.

ftctechnh committed:
update the gradle files to support newer Android Studio 2.1.x environment.

ftctechnh committed:
updated README file.

rgatkinson committed:
make compileable with current SDK


Former-commit-id: dac0f82e0c62668866574a8feef69094c928ec6d
Former-commit-id: 43ac2f591a4e44199c031fb456133ed3b8d777a2

rgatkinson committed:
make compileable with current SDK

rgatkinson committed:
fix decoding status problem


Former-commit-id: fb1c569e43cd3c30acf3320ae54af18395935d14
Former-commit-id: 6cca5841fcf5c26c603cd489f0c1191be49165fb

rgatkinson committed:
fix decoding status problem

rgatkinson committed:
pre-move FtcRobotControllerActivity; delete old SDK examples

rgatkinson committed:
pre-move FtcRobotControllerActivity; delete old SDK examples


Former-commit-id: 27f48e171ed30b9cabd966d6b637050da38c2038
Former-commit-id: 2ab12b29cc827fc6b731f747ab75a7321a707c3b

rgatkinson committed:
merge new SDK; remove TelemetryDashboardAndLog; make compile

rgatkinson committed:
merge new SDK; remove TelemetryDashboardAndLog; make compile


Former-commit-id: 43cb9965219314fbce7cf53635de205fb65eb8a6
Former-commit-id: 64709db7dacafea1240b76700f5ef9cc733ab7fa

rgatkinson committed:
remove hardware factory, as we no longer munge the hardware map; deprecate SynchronousOpMode


Former-commit-id: cd2ce44f071d52619badf2c6be4f62891c341538
Former-commit-id: c30e80a34e89811f9cfa8c214f77f01c3cb4e316

rgatkinson committed:
remove hardware factory, as we no longer munge the hardware map; deprecate SynchronousOpMode

rgatkinson committed:
update to latest SDK


Former-commit-id: f427b56e3aba3bd17e6bff5de37c6ac6363c3045
Former-commit-id: 1514f3add1040d358d87cfee4e1c01d55e8b599f

rgatkinson committed:
update to latest SDK

rgatkinson committed:
remove unneed libs copy


Former-commit-id: e0825a2e860d345908f020b518412307b7496ee0
Former-commit-id: 5c2410377d8ec03f01c1fc406d91f18c1147ceca

rgatkinson committed:
remove unneed libs copy

rgatkinson committed:
remove now-dead code


Former-commit-id: ee9a1e4971b74bc6a71d0d5fd51f544fa05d6341
Former-commit-id: 4237a88986cec392e7bfeb1aaaace31c46420eba

rgatkinson committed:
remove now-dead code

rgatkinson committed:
more dead code removal


Former-commit-id: d832f9c46dfe55962703efca8859f649c4919597
Former-commit-id: f4999233617b0df740e11e14abd88121dce3e5ae

rgatkinson committed:
more dead code removal

rgatkinson committed:
remove more dead code

rgatkinson committed:
remove more dead code


Former-commit-id: 9a20e00c08c5bf9a1ffc0704942039348547f869
Former-commit-id: 375f36c448945650d8ff653a82c7f5022e068cd2

rgatkinson committed:
Merge branch 'SwerveDev/releasePrep2' of https://github.com/SwerveRobotics/ftc_app.git

rgatkinson committed:
remove old sounds

ftctechnh committed:
v 2.0 (released on 20160819).

ftctechnh committed:
restored gradlew and gradlew.bat which were accidentally removed in the push of v 2.0

ftctechnh committed:
update gradlew and gradlew.bat

rgatkinson committed:
Track latest HW beta

rgatkinson committed:
fix merge issues

rgatkinson committed:
Update README.md

ftctechnh committed:
Release v2.1 (160903).

rgatkinson committed:
merge ftctechnh/ftc_app v2.1

rgatkinson committed:
remove Adafruit IMU

rgatkinson committed:
remove unused TCS34725 color sensor implementation (there's a near duplicat in the core SDK, though that's also currently unused).

rgatkinson committed:
update reame

rgatkinson committed:
Merge: Update to track FTC SDK v2.1

rgatkinson committed:
Update README.md

rgatkinson committed:
Update README.md

rgatkinson committed:
Update README.md
